python/py-buildbot/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-buildbot
+version             2.3.1
+revision            0
+
+maintainers         nomaintainer
+description         The Continuous Integration Framework
+long_description    The BuildBot is a system to automate the compile/test \
+                    cycle required by most software projects to validate \
+                    code changes.
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://buildbot.net/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             GPL-2
+
+checksums           sha256  cf35d2a87b11aba29c59e47e843deba1be9649114e792e830252ed8f7c35f963 \
+                    rmd160  f6136aa647589b40520db1383d05ff477324e2b9 \
+                    size    3138465
+
+python.versions     36 37
+
+if {${name} ne ${subport}} {
```

Comment:
Following my commit above about the naming of the port, this also means you need to decide on a single version of Python. This eliminates the need for the subport check.

```
python.default_version 37

depends_build-append ...
```

Reply:
Let's commonly decide about the naming scheme first.

Independent of that I wanted to suggest to stick with 3.7 only. This might again be something for @Korusuke's project to consider. Do we want or need to install multiple versions by default when creating new ports?

Reply:
We had a short [discussion](https://github.com/macports/upt-macports/issues/6) on this and it was decided as of now that as the information that is received from pypi is not reliable so we will keep the default python versions as 27 36 37 (yet to be updated), as of now upt only gives python version as 37.

Reply:
Do you want me to update the version to 37 only?

Reply:
Yes, I would do that as I see no good reason for having to support 3.6 when 3.7 works, at least not for new ports. The 3.6 is mostly there because not all the ports migrated to 3.7 yet, and before they all migrate, we cannot really mass-remove support for 3.6 either.